chips/e310x/src/chip.rs

```diff
@@ -76,35 +75,32 @@ impl<A: 'static + Alarm<'static>> kernel::Chip for E310x<A> {
     }
 
     fn service_pending_interrupts(&self) {
-        let mut reenable_intr = FieldValue::<u32, mie::Register>::new(0, 0, 0);
```

Comment:
This change also needs to happen to OpenTitan, which uses the same logic.

Reply:
Let's make that a separate PR?

Reply:
Fine with me, as long as it doesn't get lost. They are pretty much exactly the same code.